### PR TITLE
Use sdk/tests when syncing from upstream Khronos WebGL tests

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21519)

### CODE REVIEW

File: tests/wpt/webgl/meta/conformance/canvas/buffer-offscreen-test.html.ini [hunk 1]
```diff
@@ -1,6 +1,4 @@
 [buffer-offscreen-test.html]
-  bug: https://github.com/servo/servo/issues/21132
```
Comment (jdm) [inline tests/wpt/webgl/meta/conformance/canvas/buffer-offscreen-test.html.ini]: We're losing a number of annotations in this commit :/

**Comment by jdm** [comment]
What was the motivation for moving the tests from tests/mozilla/tests/webgl to tests/webgl?

**Comment by nox** [comment]
My first motivation was that the pretty deep nesting was a papercut when I am running `mach` or `git` commands, and then that we can now make a clearer difference between Servo-specific WebGL tests and Khronos WebGL tests.

File: tests/wpt/webgl/meta/conformance/canvas/buffer-offscreen-test.html.ini (hunk 1 above)
Comment (nox) [reply tests/wpt/webgl/meta/conformance/canvas/buffer-offscreen-test.html.ini]: I added them back.

**Comment by nox** [comment]
The failures are just #20734, everything else pass.

**Pull request merged**